lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_send_app_size_metrics.rb

```diff
@@ -54,26 +58,79 @@ def self.run(params)
-        raise
-      end
+      #####################################################
+      # @!group Small helper methods
+      #####################################################
+      class << self
+        # @raise if `bundletool` can not be found in `$PATH`
+        def check_bundletool_installed!
+          Action.sh('command', '-v', 'bundletool', print_command: false, print_command_output: false)
+        rescue StandardError
+          UI.user_error!('`bundletool` is required to build the split APKs. Install it with `brew install bundletool`')
+          raise
+        end
 
-      def self.find_apkanalyzer_binary
-        sdk_root = ENV['ANDROID_SDK_ROOT'] || ENV['ANDROID_HOME']
-        if sdk_root
-          pattern = File.join(sdk_root, 'cmdline-tools', '{latest,tools}', 'bin', 'apkanalyzer')
-          apkanalyzer_bin = Dir.glob(pattern).find { |path| File.executable?(path) }
+        # The path where the `apkanalyzer` binary was found, after searching it:
+        #  - in priority in `$ANDROID_SDK_ROOT` (or `$ANDROID_HOME` for legacy setups), under `cmdline-tools/latest/bin/` or `cmdline-tools/tools/bin`
+        #  - and falling back by trying to find it in `$PATH`
+        #
+        # @return [String,Nil] The path to `apkanalyzer`, or `nil` if it wasn't found in any of the above tested paths.
+        #
+        def find_apkanalyzer_binary
+          sdk_root = ENV['ANDROID_SDK_ROOT'] || ENV['ANDROID_HOME']
+          if sdk_root
+            pattern = File.join(sdk_root, 'cmdline-tools', '{latest,tools}', 'bin', 'apkanalyzer')
+            apkanalyzer_bin = Dir.glob(pattern).find { |path| File.executable?(path) }
+          end
+          apkanalyzer_bin || Action.sh('command', '-v', 'apkanalyzer', print_command_output: false) { |_| nil }
         end
-        apkanalyzer_bin || Action.sh('command', '-v', 'apkanalyzer', print_command_output: false) { |_| nil }
-      end
 
-      def self.find_apkanalyzer_binary!
-        apkanalyzer_bin = find_apkanalyzer_binary
-        UI.user_error!('Unable to find `apkanalyzer` executable in `$PATH` nor `$ANDROID_SDK_ROOT`. Make sure you installed the Android SDK Command-line Tools') if apkanalyzer_bin.nil?
-        apkanalyzer_bin
+        # The path where the `apkanalyzer` binary was found, after searching it:
+        #  - in priority in `$ANDROID_SDK_ROOT` (or `$ANDROID_HOME` for legacy setups), under `cmdline-tools/latest/bin/` or `cmdline-tools/tools/bin`
+        #  - and falling back by trying to find it in `$PATH`
+        #
+        # @return [String] The path to `apkanalyzer`
+        # @raise [FastlaneCore::Interface::FastlaneError] if it wasn't found in any of the above tested paths.
+        #
+        def find_apkanalyzer_binary!
+          apkanalyzer_bin = find_apkanalyzer_binary
+          UI.user_error!('Unable to find `apkanalyzer` executable in `$PATH` nor `$ANDROID_SDK_ROOT`. Make sure you installed the Android SDK Command-line Tools') if apkanalyzer_bin.nil?
```

Comment:
Nitpick: I can't point my finger on it, but that "nor" sounds odd. I usually see "nor" in "neither ... nor" or sentences. I think, with "unable" being a negative connotation verb, it works, but I'd suggest: 

```suggestion
        UI.user_error!('Unable to find `apkanalyzer` executable in either `$PATH` or `$ANDROID_SDK_ROOT`. Make sure you installed the Android SDK Command-line Tools') if apkanalyzer_bin.nil?
```

<img width="945" alt="image" src="https://user-images.githubusercontent.com/1218433/170431858-4ddcfcca-2d3a-4eae-bf00-062e319e7e98.png">

Reply:
Yeah I agree, tbh I found it a bit odd-looking as well when I wrote it myself 😛

Should it be "in neither $PATH nor $ANDROID_SDK_HOME" then, given it's a negative form?

Reply:
I think "Unable ... in neither ... nor ..." would make the sentence a [double negative](https://examples.yourdictionary.com/examples-of-double-negatives.html) and while they're not wrong, they "should be avoided".

But... we're hair splitting, really.